Description: Work around gpgme-config removal in new version of libgpgme-dev
 /usr/bin/gpgme-config is removed in libgpgme-dev 1.18.0-2. Use pkgconf
 to work around this. Version 0.11 of this crate removed this usage.
Last-Update: 2022-11-15
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/build.rs
+++ b/build.rs
@@ -31,7 +31,7 @@
 }
 
 fn try_config<S: Into<OsString>>(proj: &Project, path: S) -> Result<Config> {
-    let path = path.into();
+    let path: OsString = "pkgconf gpgme".into();
     let mut cmd = path.clone();
     cmd.push(" --version");
     let version = output(Command::new("sh").arg("-c").arg(cmd))?;
@@ -39,7 +39,7 @@
     let mut cmd = path;
     cmd.push(" --cflags --libs");
     if cfg!(unix) {
-        cmd.push(" --thread=pthread");
+        //cmd.push(" --thread=pthread");
     }
     proj.try_config(Command::new("sh").arg("-c").arg(cmd))
         .map(|mut cfg| {
